Conch/lib/Conch/Model/Validation.pm

```diff
@@ -96,19 +97,32 @@ Lookup a validation by ID
 
 sub lookup ( $class, $id ) {
 	my $ret =
-		Conch::Pg->new->db->select( 'validation', undef,
+		Conch::Pg->new->db->select( 'validation', $attrs,
```

Comment:
Reset back to use $attrs.

Using `undef` returns fields not purposely exposed in the module. These fields are stored in the internal blessed hashref. These fields cause `is_deeply` tests to fail when comparing whether an object created with `->create` matches an object created with `->lookup`.

I like the semantics of specifying the attributes on each query. It's decouples the model from the representation in the database. The schema may change, even be split across multiple tables, and the attributes can remain the same.

Reply:
Ah. So `Mojo::Base` is annoyingly minimal here. Over in `Moo` land, if you call `new()` with random attributes, they get ignored.